### ISSUE TITLE
Description field in  Zenodo publication

### ISSUE THREAD

**Issue by hrzepa** [ImperialCollegeLondon/hpc_portal#58]
Currently appears as Gaussian 16 (C01) calculation

Could it instead be the contents of the description field in the portal?

**Comment by rse-ci-imperial** [comment]
The description is currently used for the title of the deposition (as with the IC repository).

**Comment by cc-a** [comment]
The description is currently used for the title of the deposition (as with the IC repository).

**Comment by hrzepa** [comment]
Yes, but the title is not very useful being entirely generic?  could the two fields be transposed, so that Gaussian C  becomes the description?

**Comment by cc-a** [comment]
At the moment the description from the portal should be the title for the publication and the description of the publication should be "Gaussian 16 (C01) calculation". That seems like the right way round to me otherwise the title is generic.
